webview-ui/src/i18n/locales/en/kilocode.json

```diff
@@ -253,6 +253,20 @@
 				"label": "Chat Input Autocomplete",
 				"description": "When enabled, Kilo Code will suggest completions as you type in the chat input. Press Tab to accept suggestions."
 			},
+			"snooze": {
+				"label": "Snooze Autocomplete",
+				"description": "Temporarily pause autocomplete suggestions for a specified duration.",
+				"button": "Snooze",
+				"unsnooze": "Unsnooze",
+				"currentlySnoozed": "Autocomplete is currently snoozed.",
+				"duration": {
+					"1min": "1 minute",
+					"5min": "5 minutes",
+					"15min": "15 minutes",
+					"30min": "30 minutes",
```

Comment:
ok this felt a bit redundant, but simplyfying doesn't work with all supported languages